example/component/nav.css

```diff
@@ -1,15 +1,15 @@
 nav {
   --data: json({
     "name": ["one", "two", "three"],
-    "link": ["#one", "#two", "#three"]
+    "link": [] /* This array is filled with content in --prepare */
```

Comment:
If this is autogenerated (through assignment rather than appending), what's the point in creating an empty array? Why not just leave it out at this point?

Reply:
I left it in for clarity.

Reply:
JSON doesn't support comments, it's causing a build error

Reply:
The comment is ignored by the CSS parser.

Reply:
In chrome yes, but not in Firefox